lib/configure.js

```diff
@@ -175,124 +174,74 @@ function configure(appName, opts, dtProbes) {
  * Configure the logger based on the configuration data
  *
  * @param {String} appName: Required. The name of the application
- * @param {Object} bunyanCfg: Required. The bunyan configuration data
- * @param {arrayOfBool} verbose: Required. Array of boolean values that
+ * @param {Object} bunyanCfg: Optional. The bunyan configuration data
```

Comment:
Why are these now optional?

Reply:
They were actually always optional, but they had just been mislabeled. You could omit the bunyan config and the logging would just default to `STDERR` and the verbosity flags were usually just for command line invocation when developing or debugging.